src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,72 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+
+    public int calculate(String inputString){
```

Comment:
I believe there should be better name for things like "4 + 5 - 1" than `inputString` - this name tells nothing about purpose of the var.